### ISSUE TITLE
Quick install steps fail on "No package 'libffi' found" in ubuntu 14.04 LTS

### ISSUE THREAD

**Issue by tchalvak** [lektor/lektor#61]
Tries and fails to find a crypto package:

```
  Running setup.py install for cryptography
    Package libffi was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libffi.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libffi' found
    Package libffi was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libffi.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libffi' found
    Package libffi was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libffi.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libffi' found
    Package libffi was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libffi.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libffi' found
    Package libffi was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libffi.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libffi' found
    c/_cffi_backend.c:13:17: fatal error: ffi.h: No such file or directory
     #include <ffi.h>
```

More complete gist here:
https://gist.github.com/tchalvak/a230f8b759875748240d

**Comment by rurtle** [comment]
Please check my response [here](https://gist.github.com/tchalvak/a230f8b759875748240d). If the hints mentioned there helps in resolving it, close this issue unless you want relevant dependencies to be resolved in build script itself. In that case, you should create separate issues targeting different platforms.

**Comment by niijv** [comment]
Have a look at the requirements for ubuntu in the [official docs](https://www.getlektor.com/docs/installation/). 
Does the installation work if you get them first?

**Comment by rurtle** [comment]
@niijv While following the installation guide, I missed the imagemagick installation part. I followed the installation guide for **Development Version** & resolved dependencies as I came across. Perhaps this should be included in the steps mentioned in the **Development Version** as well or we should have a pre-requisites section applicable for all platforms, in the [official installation guide](https://www.getlektor.com/docs/installation/) IMO.

**Comment by tchalvak** [comment]
First off, I got an install working just now when I went back to it.

The install steps here:
https://github.com/lektor/lektor/blob/master/README.md
did not work either before -or even after- the install steps from the main install doc.  That's just FYI, I probably won't pursue it further, I just wanted to be able to run lektor.

What did work is the install steps from the official docs.
https://www.getlektor.com/docs/installation/

Thanks,
--Roy
